src/ActionList/Item.tsx

```diff
@@ -207,6 +207,7 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
           ref={forwardedRef}
           sx={merge<BetterSystemStyleObject>(styles, sxProp)}
           data-variant={variant === 'danger' ? variant : undefined}
+          {...(containerProps.role === 'option' && {'aria-selected': selected})}
```

Comment:
Small suggestion for clarity:

```suggestion
          aria-selected={containerProps.role === 'option' ? selected : undefined}
```

This format could make it more clear what we're setting and when along with avoiding the object spread / spread of the false-y value if `role` is not `option`

Reply:
Ah, yeah, that's better. I explicitly don't want `aria-selected` to get passed at all if the role isn't `option`.